model/godleyTab.h

```diff
@@ -28,7 +28,7 @@ namespace minsky
   class GodleyTab: public ItemTab
   {	  
   public:
-    bool itemSelector(ItemPtr i) override {if (dynamic_cast<GodleyIcon*>(i.get())) return true; return false;}    
+    bool itemSelector(ItemPtr i) override {if (dynamic_cast<GodleyIcon*>(i.get())) return i.get(); return false;}
```

Comment:
I was thinking more `return dynamic_cast<GodleyIcon*>(i.get());` The if construct is redundant.

Reply:
Done, thank you :)